docs/application/dotnet/guides/webview/overview.md

```diff
@@ -2,6 +2,9 @@
 
 You can access web pages and web content in your application using the WebView functionality. You can also use various other features for Web browsing, such as loading and displaying web pages, and navigating through the browsing history.
 
+> [!NOTE]
+> WebView class is Deprecated since API Level 10, and will be removed in API Level 12. The WebView provided by .NET MAUI or Tizen.NUI can be used instead.
```

Comment:
```suggestion
> WebView class is deprecated since API Level 10 and will be removed in API Level 12. The WebView provided by .NET MAUI or Tizen.NUI can be used instead.
```